### PR TITLE
Estimate outputs

### PR DESCRIPTION
**Pull request by SEAJang**

### CODE REVIEW

File: LDAR_Sim/src/initialization/leaks.py [hunk 1]
```diff
@@ -57,6 +57,7 @@ def generate_leak(program, site, start_date, leak_count, days_active=0, day_ts_b
         'days_active': days_active,
         'volume': None,
         'estimated_volume': None,
+        'estimated_volume_b': None,
```
Comment (KeeganShaw-GIS) [inline LDAR_Sim/src/initialization/leaks.py:60]: Whats this ?

**Review by KeeganShaw-GIS — APPROVED** [review]
Nice, Maybe add some more detail on what the different emissions estimates are and maybe include a use case.

File: LDAR_Sim/src/initialization/leaks.py (hunk 1 above)
Comment (SEAJang) [reply LDAR_Sim/src/initialization/leaks.py:60]: Estimated_volume and estimated_volume_b are used to keep track of the estimated emission rate using the method of 1/2 of time from last survey. It's to allow for people to keep track of 2 different groups of site's estimated emissions. Say you had 2 separate state/province that both require you to keep track of this emissions but eventually the calculations need to be different.

File: USER_MANUAL.md [hunk 2]
```diff
@@ -591,7 +664,7 @@ If enabled, the leaks will be stored locally in /inputs/generation after running
 
 **Notes on acquisition:** N/A
 
-**Notes of caution:** None
+**Notes of caution:** Can also be provided in the subtype_file
```
Comment (Brendanm44) [inline USER_MANUAL.md:667]: Consistent formatting please and thank you BYE

File: USER_MANUAL.md [hunk 3]
```diff
@@ -1209,7 +1343,7 @@ The follow-up delay parameter can be set to require multiple measurements for a
 
 **Notes on acquisition:** No data acquisition required.
 
-**Notes of caution:** Can be overwritten if consider_daylight is True and days are short.
+**Notes of caution:** Can be overwritten if consider_daylight is True and days are short. Until internal code is changed, max work day is 23 hours.
```
Comment (Brendanm44) [inline USER_MANUAL.md:1346]: Expand on "short" in regards to parameter lengths

**Review by Brendanm44 — APPROVED** [review]
Great job everyone :)

**Pull request merged**